src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -2982,7 +2987,6 @@ def test_vm_create_existing_nic(self, resource_group):
         self.cmd('network public-ip create -g {rg} -n my-pip')
         self.cmd('network vnet create -g {rg} -n my-vnet --subnet-name my-subnet1')
         self.cmd('network nic create -g {rg} -n my-nic --subnet my-subnet1 --vnet-name my-vnet --public-ip-address my-pip')
-        self.cmd('network nic ip-config create -n my-ipconfig2 -g {rg} --nic-name my-nic --private-ip-address-version IPv6')
```

Comment:
why remove?

Reply:
The tests fail during execution of this command (`az 'network nic ip-config create`). The test is successful on the docker container for the test run where it fails. However we see in the live logs that network responds with the minimum api issue during this command. See [here](https://dashboard.azclitest.com/task/511482). I suspect the failure is from the network RP's, maybe because of the IPv6 private address.

Also there is no need trying to create a secondary ip configuration because an ip configuration is created when the nic is created, and the point of the test is to create a VM from an existing nic.

Reply:
If the test fails on that command, it should not be an acceptable solution to simply remove it.